README.md

```diff
@@ -10,97 +10,106 @@
 ![](https://img.shields.io/badge/npm-%3E%3D6.0.0-orange.svg?style=flat-square)
 ![](https://img.shields.io/badge/Node.js-%3E%3D10.0.0-orange.svg?style=flat-square)
 
-> Simple boot node for RIF Communications PubSub
+# RIF Communications bootnode
 
-The aim to provide a libp2p node which serves as bootstrap node for projects which use [`rif-commmunications-pubsub`](https://github.com/rsksmart/rif-communications-pubsub). 
+## Getting started:
 
-This can be also used for local development where you can define the list of Rooms that will be listened on and messages printed out to STDOUT.
+- Node 14+ (14.13 recommended)
 
-## Table of Contents
+### Get the code
 
-- [RIF Communications PubSub node](#rif-communications-pubsub-node)
-  - [Table of Contents](#table-of-contents)
-  - [Usage](#usage)
-  - [Config](#config)
-  - [Docker](#docker)
-  - [License](#license)
+Download the code by executing `git clone https://github.com/rsksmart/rif-communications-pubsub-bootnode -b grpc-api`.
 
-## Usage
+After this, run `npm install` inside the cloned folder.
 
-Example of usage:
 
-```
-npm start
-```
+### Create a key for the RIF Communications bootnode
 
-Spawns a new libp2p node with new PeerId listening to TCP connections on port 6030.
+1. Create the `config/keys/lumino` folder.
+2. Execute the following commands inside this folder:
 
 ```
-NODE_ENV=develop npm start
-```
-
-Spawns a new libp2p node with PeerId `QmbQJ4FyVBAar7rLwc1jjeJ6Nba6w2ddqczamJL6vTDrwm` listening to websocket connections on port 6030 and joins rooms `0xtestroom` and `0xtestroom2`. Any peers joining and leaving the room will be logged as well as any messages in th following format:
+openssl ecparam -genkey -name secp256k1 -out ecs_key.pem -param_enc explicit
+``` 
+and then
 
 ```
-<roomName>: peer <peerId> joined
-<roomName>: peer <peerId> left
-<roomName>: message {from: <peerId>, data: <content of the message>}
-<roomName>: message {from: <peerId>, data: <content of the message>, to: <peerId>} // Only for direct messages
+openssl pkcs8 -in ec_key.pem -topk8 -v2 aes-256-cbc -v2prf hmacWithSHA256 -outform DER -out ec_key_pkcs8_v2.der
+``` 
+The latter one will require you to define a password.
+
+### Edit the private key configuration
+
+- go back to `config` folder
+- edit the `lumino.json5` file. Modify the `key.password` value to your key's password
```

Comment:
despite PR #33, i think we should probably change this. we shouldn't need to make a direct reference to `lumino` in these configs.

i propose we keep the `lumino.json5` around, but also add another base config (`client.json5`) without the bootnotes, and then make a reference to it in these sections.

Reply:
Agree. I'd keep lumino.json5 but also add client.json5. I'll change all the references to lumino.json5 to client.json5. Agree?